boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -306,7 +306,7 @@ bool App_Accumulator_CheckFaults(struct Accumulator *const accumulator, struct T
     float min_allowable_cell_temp = MIN_CELL_DISCHARGE_TEMP_DEGC;
 
     // if we are charging, max cell temp is 45C not 60C
-    if (App_TractiveSystem_GetCurrent(ts) < -3.0f)
+    if (App_TractiveSystem_GetCurrent(ts) > 3.0f)
```

Comment:
I'm not completely sure why the voltage we were checking for on the tractive system before was negative. Is it at all related to getting the ts voltage for charging finished?

Reply:
The TS current convention we want to use is that negative current is current leaving the pack (discharge), so charging current should be positive. I believe this was backwards before so that's why it had to switch